packages/web3torrent/src/clients/payment-channel-client.ts

```diff
@@ -240,7 +243,7 @@ export class PaymentChannelClient {
     }
   }
 
-  async createChannel(peers: Peers): Promise<ChannelState> {
+  async createChannel(peers: Peers, torrentHash: string): Promise<ChannelState> {
```

Comment:
One downside to this approach is that the payment-channel-client now contains torrent specific concepts. How hard would it be to store this mapping in the web3torrent client?

Reply:
Or, do you think this approach would work? https://github.com/statechannels/monorepo/pull/2161

Reply:
Totally agreed that this solution is not great. I am open to suggestions on where to store this mapping.

Reply:
The channel id to torrent info hash mapping has been moved to web3torrent library: 426d77b49.